src/components/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+// booksSlice.js
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    bookAdded(state, action) {
+      state.push(action.payload);
```

Comment:
- [ ] [OPTIONAL] in case action.type is ADD_BOOK the function returns a new state object in which the books array will contain a new book object, kindly consider the immutability of the state, so the state is not overwritten instead it should contain the previous state and the new one